### PR TITLE
add func for new test keyspace tikv store

### PR DESCRIPTION
**Pull request by ystaticy**

### CODE REVIEW

File: tikv/test_util.go [hunk 1]
```diff
@@ -93,3 +94,41 @@ func NewTestTiKVStore(client Client, pdClient pd.Client, clientHijack func(Clien
...
+		client = clientHijack(client)
+	}
+
+	if pdClientHijack != nil {
+		pdCli = pdClientHijack(pdCli)
+	}
+
+	// Make sure the uuid is unique.
+	uid := uuid.New().String()
+	spkv := NewMockSafePointKV()
```
Comment (iosmanthus) [inline tikv/test_util.go:125]: Should this spkv add a prefix too?

Comment (ystaticy) [reply tikv/test_util.go:125]: It is a `map[string]string` ，not a etcd storage at here.
So we don't need the prefix, right?

File: tikv/test_util.go [hunk 2]
```diff
@@ -93,3 +94,41 @@ func NewTestTiKVStore(client Client, pdClient pd.Client, clientHijack func(Clien
 	tikvStore.mock = true
 	return tikvStore, err
 }
+
+// NewTestTiKVStore creates a test store with Option
+func NewTestKeyspaceTiKVStore(client Client, pdClient pd.Client, clientHijack func(Client) Client, pdClientHijack func(pd.Client) pd.Client, txnLocalLatches uint, keyspaceMeta keyspacepb.KeyspaceMeta, opt ...Option) (*KVStore, error) {
```
Comment (iosmanthus) [inline tikv/test_util.go:99]: Are these test utils provided for TiDB?

Comment (ystaticy) [reply tikv/test_util.go:99]: Yes, it's for TiDB.

**Comment by iosmanthus** [comment]
Rest LGTM

**Review by iosmanthus — APPROVED** [review]

**Pull request merged**